### PR TITLE
feat(11): initail registr and pass reset forms

### PR DESCRIPTION
**Pull request by belgithub**

### CODE REVIEW

File: packages/web/src/components/SignInForm/SignInForm.test.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('components/SignInForm', () => {
     expect(container.firstChild).toMatchSnapshot();
   })
 
-  it('should render submit button disabed by default', () => {
+  it('should render submit button disabled by default', () => {
```
Comment (dubbha) [inline packages/web/src/components/SignInForm/SignInForm.test.tsx]: thanks :)

File: packages/web/src/components/SignInForm/SignInForm.tsx [hunk 2]
```diff
@@ -21,22 +22,34 @@ export const SignInForm = ({ onSubmit }: Props) => {
...
-      <Form.Control
-        type="password"
-        placeholder="Enter password"
-        value={password}
-        onChange={e => setPassword(e.target.value)}
-        data-testid="signInForm__password"
-      />
-      <SubmitButton disabled={!isValid} />
-    </Form>
+    <>
```
Comment (dubbha) [inline packages/web/src/components/SignInForm/SignInForm.tsx]: You don't need to wrap a single element into a fragment

**Review by karolinayanko — APPROVED** [review]

File: packages/web/src/components/SignInForm/SignInForm.tsx [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
 import './styles.scss';
 
+import {
+  Form,
+  SubmitButton,
+  ErrorAlert,
+  AlertPlaceholder,
+  Link,
+} from 'elements';
+
```
Comment (oleksiydubovykTR) [inline packages/web/src/components/SignInForm/SignInForm.tsx:11]: Why moving this down?

File: packages/web/src/components/SignInForm/SignInForm.tsx [hunk 4]
```diff
@@ -25,31 +32,34 @@ export const SignInForm = ({ isLoading, error, onSubmit }: Props) => {
...
+          <Link to="/reset">Forgot password?</Link>
+        </Form.Text>
       </Form.Group>
-      <SubmitButton disabled={!isValid || isLoading} />
-      <div className="register">
-        {'Don\'t have an account? '}<Link to="/signup">Register</Link>
-      </div>
+      <SubmitButton disabled={!isValid} data-testid="submitButton">
+        Sign Up
+      </SubmitButton>
```
Comment (oleksiydubovykTR) [inline packages/web/src/components/SignInForm/SignInForm.tsx:54]: Why removing the Register link?

File: packages/web/src/components/SignInForm/SignInForm.tsx [hunk 5]
```diff
@@ -25,31 +32,34 @@ export const SignInForm = ({ isLoading, error, onSubmit }: Props) => {
...
+        <Form.Text className="reset-password-link">
+          <Link to="/reset">Forgot password?</Link>
+        </Form.Text>
       </Form.Group>
-      <SubmitButton disabled={!isValid || isLoading} />
-      <div className="register">
-        {'Don\'t have an account? '}<Link to="/signup">Register</Link>
-      </div>
+      <SubmitButton disabled={!isValid} data-testid="submitButton">
+        Sign Up
```
Comment (oleksiydubovykTR) [inline packages/web/src/components/SignInForm/SignInForm.tsx]: This is Sign In

**Review by dubbha — APPROVED** [review]

**Pull request merged**